spec/Interoperation.tex

```diff
@@ -228,7 +232,11 @@ \subsubsection{Referring to External C Structs}
 a Chapel \chpl{record} definition with the \chpl{extern} keyword.
 \begin{syntax}
 external-record-declaration-statement:
-  `extern' simple-record-declaration-statement
+  `extern' external-name[OPT] simple-record-declaration-statement
+
+external-name:
+  identifier
+  string-literal
```

Comment:
I'm surprised that it's OK to have two syntax elements named 'external-name'.  I wouldn't think we'd want both.  I would've thought that the spec script that checks the graph of syntax productions for completeness would've caught this....  did it not, or did you not build the spec?